test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
```

Comment:
Nah, we are specifically avoiding `new.target` because it will blow up in non-ES6-supporting UAs